dds_web/api/user.py

```diff
@@ -43,7 +43,7 @@
 # ENDPOINTS ############################################################################ ENDPOINTS #
 ####################################################################################################
 class AddUser(flask_restful.Resource):
-    @auth.login_required
+    @auth.login_required(role=["Super Admin", "Unit Admin", "Unit Personnel", "Project Owner"])
```

Comment:
Frankly I think this is incorrect. I don't think "Project Owner" is a valid role as I've explained already.

Reply:
I have followed the agreed roles which are pinned down. We could remove that as well, if @inaod568 is fine with it. Someone can then update the agreement as well.